contents/handbook/people/compensation.mdx

```diff
@@ -70,7 +70,7 @@ The reason for less sophistication here is that we have very few executives, and
 
 We review pay proactively and currently run pay reviews for the whole team 3x per year - usually around March, July, and November. You do not need to do anything - our goal is to keep your compensation at an appropriate level without you having to ask.
 
-As we do these much more frequently than regular companies, team members should definitely not expect these to result in a change to their Step or Level each time - mostly they will stay the same. Additionally, team members will find that their Step will change more frequently than their Level. 
+As we do these much more frequently than regular companies, team members should definitely not expect these to result in a change to their Step or Level each time - mostly they will stay the same. Additionally, team members will find that their Step will change more frequently than their Level. You may receive an adjustment to your step number without changing your actual step. For example you can move from a step of 0.95 to 1.0, which increases your salary as our calculator is based on these numbers, but both of these numbers are still in the "Established" step.
```

Comment:
```suggestion
As we do these much more frequently than regular companies, team members should definitely not expect these to result in a change to their Step or Level each time - mostly they will stay the same. Additionally, team members will find that their Step, or place within a Step's range, will change more frequently than their Level.
```

Reply:
Suggested a much shorter way to clarify this! (Given that we've never had this come up as a question, I don't think it needs to be super comprehensively explained)

Reply:
Ah perfect, thanks!